src/chimpstackr_cli.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+import argparse
+import cv2
+import numpy as np
+import os
+import sys
+import tempfile
+
+parser = argparse.ArgumentParser(
+                    prog='ChimpStackr_cli',
+                    description='Command line interface to chimpstackr',
+                    epilog='Contributed by Ivo Blöchliger')
+
+parser.add_argument('-o', '--output', default="stacked.jpg", help="Name of the outputfile, must end in .jpg (or .JPG). Defaults to stacked.jpg.")
```

Comment:
What about use-cases where PNG or TIFF output is desired?
I think it should also be possible with a cli to let the user choose between all supported output file formats.